src/kubernetes/config.aat.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
Its up to you, 
There's talk of doing this for CCD so that teams can control callback urls...
I wouldn't do it unless you need it.
i.e. don't do it for the sake of it
If you have a real need for it then spike it out,

if you're doing that sort of thing it would be probably worth looking at helm charts and getting the teams to publish one that way you don't have all the config needed to run the services everywhere